externals/llvm-external-projects/torch-mlir-dialects/lib/Dialect/Tcp/IR/TcpOps.cpp

```diff
@@ -65,4 +66,9 @@ LogicalResult BroadcastOp::verify() {
   return success();
 }
 
+OpFoldResult ConstOp::fold(ArrayRef<Attribute> operands) {
```

Comment:
Is there a test for this fold?

Reply:
I encountered some issues when testing the constant folding. @navahgar (who authored this op) said he will be able to look into this sometime early next week. I believe this is the last open comment before landing the PR. In the meantime, please feel free to re-review the other changes.

Reply:
Per discussion with @navahgar , removed `hasFolder` from `Tcp_ConstOp` tablegen description, and `ConstOp::fold` for now. Noted a TODO to re-enable folding along with unit-tests.

Reply:
Great!